recipes/drupal8.php

```diff
@@ -20,3 +20,11 @@
   // https://www.drush.org/latest/deploycommand/
   invoke('drush:deploy');
 })->once();
+
+task('drush:config:export', function() {
+  $releasesList = get('releases_list');
+  $prev_release = max($releasesList);
+  $destination = '{{deploy_path}}/releases/' . $prev_release . '/config/backup';
+  run("mkdir $destination");
+  drush("config:export --destination=$destination", ['runInCurrent', 'showOutput']);
+})->once();
```

Comment:
You will want to remove the `once()` call so that this export happens on all servers (saves us from hunting down the live server at the time of the export)